src/include/sof/audio/eq_iir/iir.h

```diff
@@ -61,6 +62,50 @@ struct iir_state_df2t {
 	int64_t *delay; /* Pointer to IIR delay line */
 };
 
+/*
+ * \brief Returns the output of a biquad.
+ *
+ * 32 bit data, 32 bit coefficients and 64 bit state variables
+ * \param in input to the biquad Q1.31
+ * \param coef coefficients of the biquad Q2.30
+ * \param delay delay of the biquads Q3.61
+ */
+static inline int32_t iir_process_biquad(int32_t in, int32_t *coef,
```

Comment:
oh wait a minute - I've just noticed this in a header instead of in a math library ? I think we want to put all the common IIR, FIR etc functions in math library C files.
@johnylin76 can you and @singalsu align on with maths function should belong in the library. We can then do the function move before merging the new component PRs.

Reply:
@lgirdwood This code is outdated, in https://github.com/thesofproject/sof/pull/3263 the inline header function is removed and iir_df2t is moved to math library, which could be utilized by both eq_iir and crossover.

Reply:
oh ok great - can we close this then or will there be new updates after #3263 is merged ?